### PR TITLE
chore: automate release tagging

### PR DESCRIPTION
**Pull request by alexeagle**
Similar to https://github.com/aspect-build/rules_lint/pull/427

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 ## Releasing
 
-1. Determine the next release version, following semver (could automate in the future from changelog)
-1. Tag the repo and push it (or create a tag in GH UI)
-1. Watch the automation run on GitHub actions
+Releases are automated on a cron trigger.
+If you do nothing, eventually the newest commits will be released automatically. See .github/workflows/tag.yaml.
```
Comment (hofbi) [inline CONTRIBUTING.md:4]:
```suggestion
If you do nothing, eventually the newest commits will be released automatically. See `.github/workflows/tag.yaml`.
```

File: .github/workflows/tag.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  # Allow devs to tag manually through the GitHub UI.
+  # For example after landing a fix that customers are waiting for.
+  workflow_dispatch:
+
+  # Run twice a month, on the 1rst and 15th at 3PM UTC (8AM PST)
+  # This is a trade-off between making too many releases,
+  # which overwhelms BCR maintainers and over-notifies users,
+  # and releasing too infrequently which delays delivery of bugfixes and features.
+  schedule:
+    - cron: "0 15 1,15 * *"
```
Comment (mvukov) [inline .github/workflows/tag.yaml:17]: Shall we just set it to e.g. 20th of the month? By that date I typically have (almost) all renovate PRs merged.

**Review by mvukov — APPROVED** [review]
Hi Alex, many thanks for the contribution. I left a small comment. Otherwise LGTM.

File: .github/workflows/tag.yaml (hunk 2 above)
Comment (alexeagle) [reply .github/workflows/tag.yaml:17]: sure, feel free to make edits yourself too!

**Pull request merged**